services/src/main/java/org/fao/geonet/api/mapservers/GeoFile.java

```diff
@@ -38,14 +38,15 @@
 
 /**
  * Instances of this class represent geographic files. A geographic file can be a ZIP file including
- * ESRI Shapefiles and GeoTIFF files, or an individual GeoTIFF file.
+ * ESRI Shapefiles, GeoPackage, and GeoTIFF files, or an individual GeoTIFF file or individual GeoPackage file.
  *
  * @author Éric Lemoine, Camptocamp France SAS
  * @author Francois Prunayre
  */
 public class GeoFile implements Closeable {
     private FileSystem zipFile = null;
     private Path file = null;
+    private String _format = null;
```

Comment:
I would avoid the usage variables starting with `_`

Reply:
sure, i've just followed the style of the `_sldBody` and `_containsSld` members.

Reply:
adressed by ca91b1d